ambry-store/src/test/java/com.github.ambry.store/IndexTest.java

```diff
@@ -2332,6 +2367,68 @@ private void indexSegmentRollOverTest(short indexVersion, boolean rollOverWithPu
   }
 
   /**
+   * Tests that the index segment rolls over when there is a version change in the index value or index segment
+   * @param indexVersion the version of the index segment that will get rolled over.
+   * @param rollOverWithPutRecord {@code true} if the entry that causes rollover should be a put record,
+   *                              {@code false} if the entry that causes rollover should be a delete record
+   * @throws StoreException
+   * @throws IOException
+   */
+  /*private void indexSegmentRollOverTest(short indexVersion, boolean rollOverWithPutRecord)
```

Comment:
my bad. I was supposed to remove this commented out section. Just above, you could find this method actually.